src/database.js

```diff
@@ -44,7 +44,7 @@ function channel (pathOrRef, event = 'value') {
 
     // Returns unsubscribe function
     return () => ref.off(event, callback)
-  })
+  }, buffer)
```

Comment:
Could you add a test for this?
Something like "channel() is called with a default buffer when none is provided" (`buffers.none()`) and "channel() is called with the provided buffer".